### PR TITLE
Scheduler REST API contains a method to get the portal display properties

### PR DESCRIPTION
**Pull request by laurianed**
The properties are contained in a file whose path is located into `settings.ini` configuration file

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 7 issues

1. MINOR [Scheduler.java#L1397](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java#L1397): Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.PermissionException [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)
1. MINOR [SchedulerProxyUserInterface.java#L34](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/common/util/SchedulerProxyUserInterface.java#L34): Remove this unused import 'java.util.Properties'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [SchedulerPortalConfiguration.java#L42](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerPortalConfiguration.java#L42): Make the "logger" logger private static final. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1312)
1. MINOR [SchedulerRestInterface.java#L1874](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java#L1874): Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)
1. MINOR [SchedulerClient.java#L48](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java#L48): Remove this unused import 'java.util.Properties'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [RestSmartProxyImpl.java#L35](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java#L35): Remove this unused import 'java.util.Properties'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [RestSmartProxyImpl.java#L598](https://github.com/ow2-proactive/scheduling/blob/9816cc9a61bd9e8aa07aca0f5fe479b679e9bd39/rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java#L598): Remove this unnecessary cast to "ISchedulerClient". [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1905)

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerPortalConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    public static synchronized SchedulerPortalConfiguration getConfiguration() {
+        if (configuration == null) {
+            configuration = new SchedulerPortalConfiguration();
+        }
+        return configuration;
+    }
+
+    public String getPath() {
+        if (path == null) {
```
Comment (marcocast) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerPortalConfiguration.java]: you could initialise the path in the constructor and remove the null check in the getter method

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues

1. MINOR [Scheduler.java#L1397](https://github.com/ow2-proactive/scheduling/blob/c160be5b264a4d42df9375e953ac3f7b8dafb6a1/scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java#L1397): Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.PermissionException [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)
1. MINOR [SchedulerRestInterface.java#L1874](https://github.com/ow2-proactive/scheduling/blob/c160be5b264a4d42df9375e953ac3f7b8dafb6a1/rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java#L1874): Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues

1. MINOR [Scheduler.java#L1397](https://github.com/ow2-proactive/scheduling/blob/63eed2aed6aa95f95baa8d9aa9974edf487b5873/scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java#L1397): Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.PermissionException [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)
1. MINOR [SchedulerRestInterface.java#L1874](https://github.com/ow2-proactive/scheduling/blob/63eed2aed6aa95f95baa8d9aa9974edf487b5873/rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java#L1874): Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)

**Pull request merged**